### DISCUSSION TITLE
CHANGELOG.md Changed format

### DISCUSSION THREAD

**Discussion by rusty-snake** [libseccomp-rs/libseccomp-rs#187] [General]
Our Changed section in CHANGELOG.md can be made more user friendly in my eyes.

Currently it has the following problems:
- Silent breaking changes (i.e. not catched by compiler) are not highlighted as such.
- Having a `(**Incompatible change**)` at the end of (long) lines isn't clear
- The `(**Incompatible change**)` is used very often.

I would suggest we group those three categories like in the examples below (the new header need a better name).

---

#### Non-breaking

- foo now accepts bar

#### breaking, catched by compiler

- foo now returnes Foos enum instead of Option<bool>

#### silent breaking, not catched by compiler

- foo now does bar before baz

**Comment by ManaSugi** [comment]
@rusty-snake Thank you for the proposal.
Your idea, three categories, is really nice. I'd like to use this from v0.4.0.

**Comment by rusty-snake** [comment]
Reminder.

**Reply by ManaSugi** [reply]
Okay, let's follow this CHANGELOG format from now!

**Comment by ManaSugi** [comment]
We merged this new format to main, so close the discussion. @rusty-snake Thank you for the good proposal!!
